wp-config-local-sample.php

```diff
@@ -1,9 +1,35 @@
 <?php
+/**
+ * Local WordPress configuration setup.
+ */
 
+/**
+ * Modify database credentials for your local machine.
+ */
 define( 'DB_NAME', 'database_name_here' );
 define( 'DB_USER', 'root' );
 define( 'DB_PASSWORD', '' );
 define( 'DB_HOST', 'localhost' );
 
+/**
+ * Ensure that WP paths are appropriately routed.
+ */
 define( 'WP_SITEURL', 'http://' . $_SERVER['HTTP_HOST'] . '/wordpress' );
 define( 'WP_HOME', 'http://' . $_SERVER['HTTP_HOST'] );
+
+/**
+ * No point in running auto updates on local.
+ */
+defined( 'AUTOMATIC_UPDATER_DISABLED' ) or define( 'AUTOMATIC_UPDATER_DISABLED', true );
+
+/**
+ * Ensure that errors are visible and logged.
+ */
+defined( 'WP_DEBUG' ) or define( 'WP_DEBUG', true );
+defined( 'WP_DEBUG_LOG' ) or define( 'WP_DEBUG_LOG', true );
+defined( 'WP_DEBUG_DISPLAY' ) or define( 'WP_DEBUG_DISPLAY', true );
```

Comment:
I thought these were set in wp-config if a local config is loaded?

Reply:
Ah, you're mostly right -  `WP_DEBUG` and `SAVEQUERIES` is defined there.

I think that they might be better set in this file though since it's purely local development and it more closely follows the example given in Chassis. Loading a separate config file AND config settings in the main config file feels disjointed.

Any thoughts on moving those statements to this file only?

Reply:
Moved them over, would love your thoughts.

Reply:
Only concern is that this is a bit of a change from the existing behaviour, so might surprise people. Chassis also sets `WP_DEBUG` if it loads in a local config, but only if not explicitly turned off:

https://github.com/Chassis/Chassis/blob/master/wp-config.php#L129

Reply:
> Only concern is that this is a bit of a change from the existing behavior, so might surprise people.

This is fair that surprising people is bad, at least this should lead to improved logging behavior across projects. I'm not sure where best to document this - perhaps in the README?

> Chassis also sets WP_DEBUG if it loads in a local config, but only if not explicitly turned off

Are you saying that maybe we shouldn't set the base `WP_DEBUG` value here since Chassis sets it?